staging/src/k8s.io/legacy-cloud-providers/gce/gce.go

```diff
@@ -661,7 +661,7 @@ func (g *Cloud) Instances() (cloudprovider.Instances, bool) {
 
 // InstancesV2 returns an implementation of InstancesV2 for Google Compute Engine.
 func (g *Cloud) InstancesV2() (cloudprovider.InstancesV2, bool) {
-	return nil, false
+	return g, true
```

Comment:
This should be done in a separate PR

Reply:
Since this change is tiny, I decided to put it in a separate commit now(this is also easy to revert).
How about my decision?

Reply:
Prefer separate PR so it's more searchable and to keep the discussions specific to GCE in a separate PR thread

Reply:
ok, I'll do it.

Reply:
Reverted this change.